exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,3564 @@
+{
```

Comment:
<p>"2 sets".</p><p> </p><p>I think we should add a function to the Text AI deleting the tables. How is the user supposed to know all these names? I also found this to be very important step. I first forgot to upload the models and got the output tables full of error messages. At this point you are stuck. Even after you fix the problem the system won't be bothered to redo the failed texts.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>thats why we define the names for the customer. </p><p>i already describe the result tables below. i feel like doing it here again is not of much value. how about i add a note about the description being further down?</p>